### ISSUE TITLE
The focus issue with Neo-Tree

### ISSUE THREAD

**Issue by chaozwn** [AstroNvim/AstroNvim#2341]
### Is your feature related to a problem?

When there are multiple projects in a folder, whenever I enter one of the projects, the subdirectory always becomes the root directory. Is it possible to disable this?

### Describe the new feature

Neo-Tree does not automatically focus on subdirectories.

### Additional context

open project: 
![image](https://github.com/AstroNvim/AstroNvim/assets/51357674/1f659603-614f-42d4-ba2e-df4c91207618)

open main.rs:
![image](https://github.com/AstroNvim/AstroNvim/assets/51357674/e3421f62-bf20-4055-ab84-83bf5705ca7d)

**Comment by chaozwn** [comment]
I want to remain in the initial project directory when opening the main.rs file.

**Comment by mehalter** [comment]
Yeah this is a feature in neo-tree and unrelated to AstroNvim. Check out the neo-tree documentation: https://github.com/nvim-neo-tree/neo-tree.nvim

**Comment by mehalter** [comment]
It should be something like follow current file or something like that
